crates/ruff/src/rules/flake8_bugbear/rules/except_with_non_exception_classes.rs

```diff
@@ -0,0 +1,60 @@
+use std::collections::VecDeque;
+
+use rustpython_parser::ast::{Excepthandler, ExcepthandlerKind, Expr, ExprKind};
+
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::types::Range;
+
+use crate::checkers::ast::Checker;
+use crate::registry::Diagnostic;
+use crate::violation::Violation;
+
+#[violation]
+pub struct ExceptWithNonExceptionClasses;
+
+impl Violation for ExceptWithNonExceptionClasses {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!("`except` handlers should only be exception classes or tuples of exception classes")
```

Comment:
Tweaked this to use backticks around except which differs from bugbear but it's more consistent with how we tend to format messages.